src/ImageSharp/Properties/AssemblyInfo.cs

```diff
@@ -37,4 +37,4 @@
 // Ensure the internals can be tested.
 [assembly: InternalsVisibleTo("ImageSharp.Benchmarks")]
 [assembly: InternalsVisibleTo("ImageSharp.Tests")]
-[assembly: InternalsVisibleTo("ImageSharp.Tests46")]
+[assembly: InternalsVisibleTo("ImageSharp.Tests46")]
```

Comment:
Ooops .. I forgot to remove this from my previous PR. I will change this to `ImageSharp.Sandbox46` with #66